CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.19.2)
```

Comment:
Why 3.19.2 ? this seems a little bit overkill can we limit it to max 3.16.3 (Ubuntu 20.04) or 3.10.2 (Ubuntu 18.04) ?

src: https://repology.org/project/cmake/badges

Reply:
3.19.2 comes from native cmake host detection support : https://cmake.org/cmake/help/latest/release/3.19.html#platforms

Reply:
erf my bad so if we want to correctly support M1 we'll need it...

Reply:
yeah. I looked back that thread and it was you who posted this cmake link :)